### PR TITLE
feature/VTA-671 - Defect pages

### PR DESCRIPTION
**Pull request by FBarrassDVSA**
This PR will consist of the following pages:
image

### CODE REVIEW

**Comment by FBarrassDVSA** [comment]
advisory-details:
image

**Comment by FBarrassDVSA** [comment]
defect-details-specialist-testing:
image

**Comment by FBarrassDVSA** [comment]
test-complete:
image

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -147,6 +147,42 @@ const routes: Routes = [
...
+        .then( m => m.AdvisoryDetailsModule)
+  },
+  {
+    path: PAGE_NAMES.DEFECT_DETAILS_PAGE,
+    loadChildren: () =>
+      import('./pages/testing/defects/defect-details/defect-details.module')
+        .then( m => m.DefectDetailsModule)
+  },
+  {
+    path: PAGE_NAMES.DEFECT_DETAILS_SPECIALIST_TESTING,
```
Comment (tomcrawleyy) [inline src/app/app-routing.module.ts:182]: maybe a bit pedantic but this doesnt follow the same pattern name as the rest

**Review by tomevs88 — APPROVED** [review]

**Review by tomevs88 — COMMENTED** [review]
.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (FBarrassDVSA) [reply src/app/app-routing.module.ts]: > 

This is a screenshot from the Ionic 3 app, once you select one of the options, the next time you open it, it looks like this.
image

**Comment by FBarrassDVSA** [comment]
> 

This is a screenshot from the Ionic 3 app, once you select one of the options, the next time you open it, it looks like this.
image

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**